tests/wpt/mozilla/tests/css/font_color_attribute_change.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="match" href="font_color_attribute_change_ref.html">
+  </head>
+  <body>
+    <font color="red">I should be green</font>
+    <script>
+      document.querySelector("font").setAttribute("color", "green");
```

Comment:
Let's add `document.body.offsetTop;` before this line, to ensure that layout has happened at least once.